nginx/Dockerfile

```diff
@@ -13,8 +13,6 @@ ADD nginx/docker/supervisord.conf /etc/supervisor/conf.d/supervisord.conf
 ADD nginx/docker/php-fpm-app.conf /usr/local/etc/php-fpm.d/php-fpm-app.conf
 ADD symfony.ini /usr/local/etc/php/conf.d/symfony.ini
 
-RUN sed -i 's/user www-data;//g' /etc/nginx/nginx.conf
```

Comment:
?

Reply:
This line is always remove by this sed, and is never used, so i just removed the line out of the original file